client/jetpack-cloud/sections/partner-portal/stored-credit-card/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { PaymentLogo } from '@automattic/composite-checkout';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function StoredCreditCard( props ): ReactElement {
+	const translate = useTranslate();
+	const creditCard = props.card;
+
+	const expiry = new Date( creditCard.expiry );
```

Comment:
AFAIK Date uses the client's timezone which can mess up the expiry due to the offset changing the expiration day and potentially the month (e.g. expiry if 01/01/2022 but timezone conversion puts it at 12/31/2021).
Does Stripe not provide us with any string for the expiry so we don't have to do this ourselves? If not, we should check whether the expiration is in UTC for example and use that on our side.

Reply:
Stripe will let us fetch the card dates, so we can get the card month+year values

Reply:
>Does Stripe not provide us with any string for the expiry so we don't have to do this ourselves? 
The creditCard.expiry is already a formatted date string, I'm just gonna update the way we handle it.

Reply:
Also, `Date.getMonth()` returns the month starting at 0, so this was wrong anyway. Nice catch!